plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -353,9 +355,19 @@ export class MySQLInstrumentation extends InstrumentationBase<
 
         span.setAttribute(
           SemanticAttributes.DB_STATEMENT,
-          getDbStatement(query, format, values)
```

Comment:
Since we don't use the `format` anymore, should we remove it from the function parameters as well?